.github/workflows/build.yml

```diff
@@ -42,18 +42,16 @@ jobs:
             symfony-version: ''
 
           # Symfony jobs:
-          - php: 8.1
-            os: ubuntu-latest
-            composer-mode: update
-            symfony-version: '4.4'
           - php: 8.2
             os: ubuntu-latest
             composer-mode: update
             symfony-version: '5.4'
+
           - php: 8.2
             os: ubuntu-latest
             composer-mode: update
             symfony-version: '6.0'
```

Comment:
This should be a job using `6.4`, not `6.0`. `6.0` is not a LTS version (and not receiving fixes anymore)

Reply:
btw, we should not have a job for `7.0` either, as it is not a LTS either.